pyindi-client/migration.md

```diff
@@ -0,0 +1,343 @@
+---
+title: Migration
+nav_order: 1
+parent: PyIndi Client
+permalink: /pyindi-client/migration/
+---
+
+# Migration to INDI Core 2.0.0
+
+This documentation is only for those using an old version of INDI and now trying to migrate to the INDI Core 2.0.0
```

Comment:
If you used PyINDI prior to v2.0, you need to upgrade your project to compile with INDI 2.0.0+ architecture that utilizes INDI new properties.